src/typing_extensions.py

```diff
@@ -846,6 +849,41 @@ def __round__(self, ndigits: int = 0) -> T_co:
             pass
 
 
+if hasattr(io, "Reader") and hasattr(io, "Writer"):
+    Reader = io.Reader
+    Writer = io.Writer
+else:
+    @runtime_checkable
+    class Reader(Protocol[T_co]):
+        """Protocol for simple I/O reader instances.
+
+        This protocol only supports blocking I/O.
+        """
+
+        __slots__ = ()
+
+        @abc.abstractmethod
+        def read(self, size: int = ..., /) -> T_co:
+            """Read data from the input stream and return it.
+
+            If size is specified, at most size items (bytes/characters) will be
```

Comment:
I'd either use italics or backticks here. Don't have a strong opinion about which; I think it will render fine either way in VSCode-generated tooltips etc.!

```suggestion
            If *size* is specified, at most *size* items (bytes/characters) will be
```